pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md

```diff
@@ -2,207 +2,643 @@
+In your local version of the Python bot repo, create a file called `metricity-config.toml` and insert the following lines:
+```yaml
+[bot]
+guild_id = replace_with_your_guild_id
 ```
+To properly replicate production behavior, set the `staff_role_id`, `staff_categories` and `ignore_categories` fields as well.
 
-## Run on the host
-
-Running on the host is particularly useful if you wish to debug the bot.
-The site has to have been started somehow beforehand.
-
-```shell
-poetry run task start
-```
+Now, `docker-compose up` will also start Metricity.
 
----
-## Working with Git
-Now that you have everything setup, it is finally time to make changes to the bot!
-If you have not yet [read the contributing guidelines](../contributing-guidelines), now is a good time.
-Contributions that do not adhere to the guidelines may be rejected.
+If you want to run the bot locally, you can run `docker-compose up metricity` instead.
 
-Notably, version control of our projects is done using Git and Github.
-It can be intimidating at first, so feel free to ask for any help in the server.
+### Issues?
+If you have any issues with setting up the bot, come discuss it with us on the [#dev-contrib](https://discord.gg/2h3qBv8Xaa) channel on our server.
 
-[**Click here to see the basic Git workflow when contributing to one of our projects.**](../working-with-git/)
+If you find any bugs in the bot or would like to request a feature, feel free to open an issue on the repository.
 
-## Adding new statistics
+### Appendix: Full ENV File Options
+The following is a list of all available environment variables used by the bot:
 
-Details on how to add new statistics can be found on the [statistic infrastructure page](https://blog.pythondiscord.com/statistics-infrastructure).
-We are always open to more statistics so add as many as you can!
+| Variable | Required | Description |
+| -------- | -------- | -------- |
+| `BOT_TOKEN` | Always | Your Discord bot account's token (see [Set Up a Bot Account](##set-up-a-bot-account)). |
+| `BOT_API_KEY` | When running bot without Docker | Used to authenticate with the site's API. When using Docker to run the bot, this is automatically set. By default, the site will always have the API key shown in the example below. |
+| `BOT_SENTRY_DSN` | When connecting the bot to sentry | The DSN of the sentry monitor. |
+| `BOT_TRACE_LOGGERS ` | When you wish to see specific or all trace logs | Comma separated list that specifies which loggers emit trace logs through the listed names. If the ! prefix is used, all of the loggers except the listed ones are set to the trace level. If * is used, the root logger is set to the trace level. |
+| `BOT_DEBUG` | In production | `true` or `false`, depending on whether to enable debug mode, affecting the behavior of certain features. `true` by default.
+| `REDIS_PASSWORD` | When not using FakeRedis | The password to connect to the redis database. *Leave empty if you're not using REDIS.* |
```

Comment:
what happens if you don't but still set `fakeredis=true`?

Reply:
It will have no effect. I guess it's not clear enough